ngboost/tests/test_distns.py

```diff
@@ -12,6 +12,10 @@
     LogNormal,
     Normal,
     k_categorical,
+    T,
+    TFixedDF,
```

Comment:
This is failing on import. TFixedDF would need to be in it's own file for this type of import.

Reply:
@ryan-wolbeck how would we make the import work like this?

Reply:
Please correct me, but I think this import is actually fine - the reason it's not importing is because it's just a typo, it should be `TFixedDf`, not `TFixedDF`

(capital F at the end)

Reply:
I think the imports should work now, just pushed fixes

Reply:
Good catch @mzjp2, I was reading too far into it haha